bodhi/server/buildsys.py

```diff
@@ -92,6 +93,8 @@ class DevBuildsys:
                    'tag_id': 19,
                    'task_id': 127621}
 
+    _side_tag_re = re.compile(r'(?P<base>.*)-side-(?P<id>[0-9]+)')
```

Comment:
I'm not seeing this being used anywhere, am I missing something?

Reply:
Good catch :-) removed